### PR TITLE
added link by node column analysis

### PR DESCRIPTION
**Pull request by javisantana**
cc @javierarce

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/column-options.js [hunk 1]
```diff
@@ -19,6 +24,7 @@ module.exports = Backbone.Model.extend({
   },
 
   filterByType: function (columnType) {
+    if (!this._querySchemaModel) return [];
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/column-options.js]: I think that this change should be added to the `all` method too.

File: lib/assets/javascripts/cartodb3/components/modals/add-analysis/analysis-option-models/weight-analysis-option-model.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var AnalysisOptionModel = require('./analysis-option-model');
+
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/components/modals/add-analysis/analysis-option-models/weight-analysis-option-model.js]: I think this file is not needed. I'm not sure what is it :/

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/column-options.js (hunk 1 above)
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/column-options.js]: it calls filterByType at the end, rigth?

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-by-node-column.tpl [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<form>
+  <div class="Editor-HeaderInfo">
+    <div class="Editor-HeaderNumeration CDB-Text is-semibold u-rSpace--m">2</div>
+    <div class="Editor-HeaderInfo-inner CDB-Text" data-fields="source,filter_source">
+      <div class="Editor-HeaderInfo-title u-bSpace--m">
+        <h2 class="CDB-Text CDB-HeaderInfo-titleText CDB-Size-large"><%- _t('analyses.cluster') %></h2>
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-by-node-column.tpl]: `analyses.cluster` should be `analyses.filter-by-node-column` and then an entry should be added to `en.json`.

**Comment by javierarce** [comment]
I added some comments. I'd also recommend to bump the frontend version in the packajge.json before merging. 

Besides those things, :+1:

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/column-options.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/column-options.js]: Oh, true! Nevermind then :)

**Pull request merged**